patches/minecraft/net/minecraft/world/inventory/BrewingStandMenu.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/world/inventory/BrewingStandMenu.java
 +++ b/net/minecraft/world/inventory/BrewingStandMenu.java
+@@ -13,6 +_,7 @@
+ import net.minecraft.world.item.alchemy.PotionUtils;
+ 
+ public class BrewingStandMenu extends AbstractContainerMenu {
++   private static final net.minecraftforge.common.BrewingFuelValues BREWING_FUELS = net.minecraftforge.common.ForgeHooks.onBrewingFuelRegisterEvent();
```

Comment:
This fires the event during the static initializer of the brewing stand menu, this is not a good place for that. A better place would be to make it part of the registry events, Or perhaps instead of making it a custom registry implementation. It could be data driven and use the syncing mechanics of the custom data registries?

Reply:
I'll look into those and see if I can find a better way. One thing, though: you mentioned [on Discord](https://discord.com/channels/1129059589325852724/1129069883813007360/1293357425344122930) that using tags would not be the best idea - wouldn't making it data driven have the same problems with vanilla server/Forge client?

Reply:
Quite possibly, it would need testing and a way to verify that connecting to vanilla servers behaves fine.